deepfence_worker/cronjobs/notification.go

```diff
@@ -31,62 +36,39 @@ func SendNotifications(msg *message.Message) error {
 	}
 	integrations, err := pgClient.GetIntegrations(ctx)
 	if err != nil {
-		log.Error().Msgf("Error getting postgresCtx: %v", err)
+		log.Error().Msgf("Error getting postgresCtx", err)
 		return nil
 	}
-	wg := sync.WaitGroup{}
+
+	wg := new(sync.WaitGroup)
 	wg.Add(len(integrations))
 	for _, integrationRow := range integrations {
-		go func(integration postgresql_db.Integration) {
-			defer wg.Done()
-			err := processIntegrationRow(integration, msg)
-			update_row := err != nil || (err == nil && integration.ErrorMsg.Valid)
-			if update_row {
-				var params postgresql_db.UpdateIntegrationStatusParams
-				if err != nil {
-					log.Error().Msgf("Error on integration %v", err)
-					params = postgresql_db.UpdateIntegrationStatusParams{
-						ID: integration.ID,
-						ErrorMsg: sql.NullString{
-							String: err.Error(),
-							Valid:  true,
-						},
-					}
-				} else {
-					params = postgresql_db.UpdateIntegrationStatusParams{
-						ID: integration.ID,
-						ErrorMsg: sql.NullString{
-							String: "",
-							Valid:  false,
-						},
-					}
-				}
-				pgClient.UpdateIntegrationStatus(ctx, params)
-			}
-		}(integrationRow)
+		go processIntegrationRow(wg, integrationRow, msg)
```

Comment:
This code was pushed recently - why is that being reverted?

Reply:
Done.